CHANGELOG.md

```diff
@@ -1,5 +1,67 @@
-# OP-TEE - version 2.0.0
+# OP-TEE - version 2.1.0
+
+## New features
+
+* New supported platform: Xilinx Zynq UltraScale+ MPSOC
+
+* GCC5 support
+
+* Non Linear Mapping support: In OP-TEE kernel mode, the physical to virtual
+  addresses was linear until this release, meaning the virtual addresses
+  were equal to the physical addresses. This is no more the case in this
+  release.
+
+* Font rendering routines have been introduced in order to ease an
+  implementation of Trusted UI.
+
+* File Storage: Possibility to use the normal world filesystem and the RPMB
+  implementations simultaneously.
 
+* [Link][github_commits_2_1_0] to a list of all commits between this and
+  previous release.
+
+
+## Tested on
+Definitions:
+
+| Type | Meaning |
+| ---- | ------- |
+| Standard tests | The [optee_test][optee_test] project. |
+| Extended tests | optee_test with tests from the GlobalPlatform™ TEE Initial Configuration Test Suite v1.1.0.4. |
+
+*	ARM Juno Board (vexpress-juno), standard.
+*	Foundation Models (vexpress-fvp), standard tests + extended tests,
+	using FVP ARM V8 Foundation Platformr0p0 (platform build 10.0.37)
+*	FSL ls1021a (ls-ls1021atwr), standard tests.
+*	HiKey (hikey), standard + extended tests.
+*	MTK8173-EVB (mediatek-mt8173), not tested.
```

Comment:
Should we put the non-tested below the rest. If you quickly look at the section header name and glance through this, you might miss the "not tested" part.